.babelrc

```diff
@@ -1,3 +1,12 @@
 {
-  "presets": ["env","es2015","stage-0"]
+  "presets": [
+    ["env", {
```

Comment:
This is like an automated way to say "please supply whatever polyfills are needed for ie 11 to not explode, given our config".